src/lang/uk.rs

```diff
@@ -399,162 +393,271 @@ const KOPIYKAS: [[&str; 6]; 2] = [
+                "рингіт{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::NOK => format!(
+                "крон{}",
+                NOUN_1ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::PEN => format!(
+                "сол{}",
+                NOUN_2ST_GROUP_SOFT_DECLENSIONS[number_idx][declination_idx]
+            ),
             Currency::PLN => {
                 let flextion = if self.number == GrammaticalNumber::Plural {
                     ADJECTIVE_HARD_FLEXIONS_PLURAL
                 } else {
                     ADJECTIVE_HARD_FLEXIONS_SINGULAR[Gender::Masculine.index()]
-                }[declenation_idx];
+                }[declination_idx];
                 format!("злот{}", flextion)
             }
-            Currency::QAR | 
-            Currency::RIYAL |
-            Currency::SAR => format!("ріал{}", NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
-            Currency::RUB => //format!("рубл{}", NOUN_1ST_GROUP_MASCULINE_SOFT_DECLENSIONS[number_idx][declenation_idx]),
-                             String::from("руській воєнний корабль іді нахуй"),
-            Currency::THB => format!("бат{}", NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
-            Currency::TRY => format!("куруш{}",NOUN_1ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
-            Currency::UAH => String::from(HRYVNIAS[number_idx][declenation_idx]),
-            Currency::VND => format!("донг{}", NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
-            Currency::ZAR => format!("ранд{}", NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declenation_idx]),
+            Currency::QAR | Currency::RIYAL | Currency::SAR => format!(
+                "ріал{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::RUB =>
+            //format!("рубл{}", NOUN_1ST_GROUP_MASCULINE_SOFT_DECLENSIONS[number_idx][declination_idx]),
+            {
+                String::from("руській воєнний корабль іді нахуй")
+            }
+            Currency::THB => format!(
+                "бат{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::TRY => format!(
+                "куруш{}",
+                NOUN_1ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::UAH => String::from(HRYVNIAS[number_idx][declination_idx]),
+            Currency::VND => format!(
+                "донг{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
+            Currency::ZAR => format!(
+                "ранд{}",
+                NOUN_2ST_GROUP_HARD_DECLENSIONS[number_idx][declination_idx]
+            ),
             //_ => currency.default_string(self.number == GrammaticalNumber::Plural),
```

Comment:
related with #21, this is supposed to be my way of doing, by putting a "default value" through a `default_string` function. using non_exhaustive is better that this, but my initial plan was to keep this uncommented in case someone in the future adds a currency.

I've also added a bunch of currencies in the past just to make sure adding new currencies isn't supposed to happen often

Reply:
I think currencies are updating more quickly.
In Ukraine, we had 3 currencies over the past 35 years (Soviet ruble, Ukrainian karbovanets, Ukrainian hryvnia), and expect Euro to be here in 10 years. There are some 180 currencies in the world; even if the average time of life of the currency is 200 year (clearly an overestimation), it still means one currency is changed in a year and 40 days.

Reply:
Anyway, I think it's a good idea to leave it there - when the new currency will be added, this will at least help to fix the build for testing other languages.